src/Functions/array/arrayEnumerateUniqRanked.cpp

```diff
@@ -30,36 +30,40 @@ It allows for enumeration of a multi-dimensional array with the ability to speci
     FunctionDocumentation::ReturnedValue returned_value = "Returns an N-dimensional array the same size as `arr` with each element showing the position of that element in relation to other elements of the same value.";
     FunctionDocumentation::Examples examples = {
         {"Example 1", R"(
-With `clear_depth=1` and `max_array_depth=1`, the result of `arrayEnumerateUniqRanked` is identical to that which [`arrayEnumerateUniq`](#arrayenumerateuniq) would give for the same array.
+/*
+With clear_depth=1 and max_array_depth=1, the result of arrayEnumerateUniqRanked
+is identical to that which arrayEnumerateUniq would give for the same array.
+*/
 
-```sql
 SELECT arrayEnumerateUniqRanked(1, [1,2,1], 1);
-```
         )", "[1,1,2]"},
         {"Example 2", R"(
+/*
+with clear_depth=1 and max_array_depth=1, the result of arrayEnumerateUniqRanked
+is identical to that which arrayEnumerateUniqwould give for the same array.
+*/
 
-With `clear_depth=1` and `max_array_depth=1`, the result of `arrayEnumerateUniqRanked` is identical to that which [`arrayEnumerateUniq`](#arrayenumerateuniq) would give for the same array.
-
-```sql
 SELECT arrayEnumerateUniqRanked(1, [[1,2,3],[2,2,1],[3]], 2);", "[[1,1,1],[2,3,2],[2]]
-```
+
 )", "[1,1,2]"},
         {"Example 3", R"(
+/*
+In this example, arrayEnumerateUniqRanked is used to obtain an array indicating,
+for each element of the multidimensional array, what its position is among elements
+of the same value. For the first row of the passed array, [1,2,3], the corresponding
```

Comment:
(Side note: we use in some places `[1,2,3]` and in some places `[1, 2, 3]`. It would be great to be consistent with whitespaces.)